src/index.js

```diff
@@ -1 +1,71 @@
-require('../vendor/aframe.min.js');
+/**
+ * @file index.js
+ * Defines the ContentaAframe react component, and renders it against #root.
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Scene, Entity } from 'aframe-react';
+
+require('aframe');
+require('./index.css');
+
+// Load paths to assets.
+const assets = {
+  models: {
+    layout: require('./assets/models/layout.dae'),
+    bowl: require('./assets/models/bowl.dae'),
+    stoveDoor: require('./assets/models/stove-door.dae')
+  },
+  images: {
+    floor: require('./assets/images/floor.jpg'),
+    skymap: require('./assets/images/sky.jpg')
+  }
+};
+
+/**
+ * Main ContentaAframe component.
+ * {@inheritdoc}
+ */
+const ContentaAframe = () =>
+  <Scene inspector="url: https://aframe.io/releases/0.4.0/aframe-inspector.min.js">
```

Comment:
Should this be 0.6.0 or 0.6.1? I see that package.json as 0.6.0 (though 0.6.1 is the latest release of aframe)